components/docs-chef-io/layouts/shortcodes/automate/automate_cli_commands.html

```diff
@@ -2,44 +2,101 @@
 {{ range $data_path }}
 {{ $id := urlize .name }}
 {{ $command := strings.TrimPrefix "chef-automate " .name }}
-  <h3 id="{{ $id }}">{{ .name }}</h2>
-  <p>{{ .description }}</p>
-
-  <h4 id="{{ $id }}-usage">Usage</h3>
-  <pre><code>{{ .usage }}</code></pre>
-
-  <h4 id="{{ $id }}-flags">Flags</h3>
-  <div class="fake-code grid-x">
-    {{- range .options -}}
-    <div class="cell medium-4">{{ if .shorthand }}-{{ .shorthand }}, {{ end }}--{{ .name }}</div>
-    <div class="cell medium-8">{{ .usage }}{{ if .default_value }} (default: {{ .default_value }}){{ end }}</div>
-    {{- end -}}
-    {{- range .inherited_options -}}
-    <div class="cell medium-4">{{ if .shorthand }}-{{ .shorthand }}, {{ end }}--{{ .name }}</div>
-    <div class="cell medium-8">{{ .usage }}{{ if .default_value }} (default: {{ .default_value }}){{ end }}</div>
-    {{- end -}}
-  </div>
-
-  <h4 id="{{ $id }}-see-also">See Also</h3>
-  <div class="grid-x">
-    {{ range .see_also }}
-    {{ $sa_bits := split . " - " }}
-    {{ $sa_id := index $sa_bits 0 }}
-    {{ $sa_string := index $sa_bits 1 }}
-
-    {{ $sa_command := "" }}
-    {{ $sa_command_id := "" }}
-
-    {{ if ( hasPrefix $id (urlize $sa_id ) )}}
+{{ if or (not .compatible_with) (eq .compatible_with "Automate") }}
+<h3 id="{{ $id }}">
+   {{ .name }}</h2>
+   <p>{{ .description }}</p>
+   <h4 id="{{ $id }}-usage">
+   Usage
+</h3>
+<pre><code>{{ .usage }}</code></pre>
+<h4 id="{{ $id }}-flags">
+Flags</h3>
+<div class="fake-code grid-x">
+   <table>
+      <tr class="table-heading">
+         <th>Flags</th>
+         <th>Description</th>
+      </tr>
+      {{- range .options -}}
+      <tr>
+         {{ if or (not .compatible_with_options) (eq .compatible_with_options "Automate") }}
+         <td>
+            <div class="cell medium-4">{{ if .shorthand }}-{{ .shorthand }}, {{ end }}--{{ .name }}</div>
+         </td>
+         <td>
+            <div class="cell medium-8">{{ .usage }}{{ if .default_value }} (default: {{ .default_value }}){{ end }}</div>
+         </td>
+         {{ end }}
+      </tr>
+      {{- end -}}
+      {{- range .inherited_options -}}
+      <tr>
+         {{ if or (not .compatible_with_options) (eq .compatible_with_options "Automate") }}
+         <td>
+            <div class="cell medium-4">{{ if .shorthand }}-{{ .shorthand }}, {{ end }}--{{ .name }}</div>
+         </td>
+         <td>
+            <div class="cell medium-8">{{ .usage }}{{ if .default_value }} (default: {{ .default_value }}){{ end }}</div>
+         </td>
+         {{ end }}
+      </tr>
+      {{- end -}}
+   </table>
```

Comment:
This should be a description list. https://developer.mozilla.org/en-US/docs/Web/HTML/Element/dl
For example, like this: https://docs.chef.io/workstation/ctl_chef/#options